.circleci/config.yml

```diff
@@ -13,6 +13,12 @@ executors:
     resource_class: medium
     environment:
       NODE_OPTIONS: --max_old_space_size=3072
+  node-linux-medium:
+    machine:
+      image: ubuntu-2204:2024.04.4
+    resource_class: large
```

Comment:
This is called `node-linux-medium` but is using `resource_class: large`
TODO: let's re-evaluate whether it makes sense to use `large`, and also whether we should be using `ubuntu-2204:2024.04.4` or `cimg/node:20.11-browsers`